third_party/bigframes_vendored/pandas/core/reshape/encoding.py

```diff
@@ -0,0 +1,64 @@
+# Contains code from https://github.com/pandas-dev/pandas/blob/main/pandas/core/reshape/encoding.py
+"""
+Encoding routines
+"""
+from __future__ import annotations
+
+from bigframes import constants
+
+
+def get_dummies(
+    data,
+    prefix=None,
+    prefix_sep="_",
+    dummy_na=False,
+    columns=None,
+    sparse=False,
+    drop_first=False,
+    dtype=None,
+):
+    """
+    Convert categorical variable into dummy/indicator variables.
+
+    Each variable is converted in as many 0/1 variables as there are
+    different values. Columns in the output are each named after a value;
+    if the input is a DataFrame, the name of the original variable is
+    prepended to the value.
+
+    Args:
+      data (array-like, Series, or DataFrame):
```

Comment:
Need to update this to say only Series and DataFrame. 

In general, check if the documentation matches the level of implementation

Reply:
Thanks for catching that. Fixed.